tools/lan_choose_character_kart/lan_choose_character_kart.py

```diff
@@ -0,0 +1,299 @@
+import itertools
+import os
+import subprocess
+import tempfile
+from dataclasses import dataclass
+
+from tools.gc_c_kit import devkit_tools
+from tools.lan_choose_character_kart import symbols
+
+
+def run_subprocess(arguments: list[str]) -> subprocess.CompletedProcess:
+    completed_process = subprocess.run(arguments, capture_output=True)
+    assert completed_process.returncode == 0,\
+    f'ERROR CODE: {completed_process.returncode},\
+ {completed_process.stdout.decode()}\n{completed_process.stderr.decode()}'
+    return completed_process
```

Comment:
This really could just be:

```suggestion
def run_subprocess(arguments: list[str]) -> str:
    return subprocess.check_output(arguments, text=True)
```

`check_output()` will raise if there was any extremely unexpected error.